lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -59,6 +59,8 @@ export class RunsOverviewModel extends OverviewPageModel {
         // Export items
         this._allRuns = RemoteData.NotAsked();
 
+        this._horizontalScrollEnabled = false;
```

Comment:
What is the purpose of this line if the variable is defined by default to `false` in the OverviewModel?